shared-roles/lusers/tasks/main.yml

```diff
@@ -2,24 +2,30 @@
 - become: yes
   tags: lusers
   vars:
+    login_shell:
+      Debian: "/bin/bash"
```

Comment:
Is there a key for "Linux"?

Reply:
there is `ansible_system`...

Reply:
... it gives results like these:
```
$ column -s, -t os.csv 
hostname  system   distribution  version  release
gleason   OpenBSD  OpenBSD       6.8      release
bite      Linux    Debian        10       buster
```

Reply:
i'll keep ansible_distribution since linux does not strictly imply gnu or even that bash is in /bin. we don't have that many distributions anyway